### PR TITLE
Add API to create a fact collection from two or more completed  ScanJobs

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: quipucords/api/messages.py [hunk 1]
```diff
@@ -108,6 +108,17 @@
 SJ_EXTRA_VARS_KEY = 'Extra vars keys must be jboss_eap, jboss_fuse, '\
     'or jboss_brms.'
 
+SJ_MERGE_JOB_REQUIRED = 'This field is required'
+SJ_MERGE_JOB_NOT_LIST = 'This field must be a list of job ids.'
+SJ_MERGE_JOB_TOO_SHORT = 'Two or more job ids are required.'
+SJ_MERGE_JOB_NOT_INT = 'Job ids must be integers.'
+SJ_MERGE_JOB_NOT_UNIQUE = 'Set of ids must be unique.'
+SJ_MERGE_JOB_NOT_FOUND = 'Not all job ids exist. Jobs note found: %s'
```
Comment (abaiken) [inline quipucords/api/messages.py]: Jobs note found --> jobs not found

File: quipucords/api/fact/util.py [hunk 2]
```diff
@@ -33,6 +33,26 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def build_sources_from_tasks(tasks):
```
Comment (chambridge) [inline quipucords/api/fact/util.py]: Not a great name. I know what you mean here, but its more like `build_fact_collection_data_from_tasks()`

File: quipucords/api/messages.py (hunk 1 above)
Comment (chambridge) [reply quipucords/api/messages.py]: let's use scan job just to be specific.... everywhere!

Comment (kholdaway) [reply quipucords/api/messages.py]: Done for both

File: quipucords/api/fact/util.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/api/fact/util.py]: Its not quite that either.

**Pull request merged**